### ISSUE TITLE
why one side is dead,the other side keep fire?

### ISSUE THREAD

**Issue by xiaoxiangzju** [mwydmuch/ViZDoom#263]
python3.5.2 Ubuntu16.04
I find sometimes one side is trapped in the blue area by the bomb vibrating,if I set the game game.set_mode(Mode.ASYNC_PLAYER)  mode,after that I find the health is not below the zero by the code game.get_game_variable(GameVariable.HEALTH ) even 100% healthy.is it normal the other side keep fire once one side is dead.again,the video sometimes miss a player in multi-player mode ,but the code tell me the player is still there.can you give some advice? thank you

**Comment by mwydmuch** [comment]
Hello @xiaoxiangzju,
excuse me, but I'm not sure, what is your issue, could you rephrase it?

**Comment by xiaoxiangzju** [comment]
python3.5 Ubuntu16.04 
I use the map multiduel.wad,Is it ok the video show the right situation as the code goes?I find use this map when one side fall into the green area(sorry about blue area) the health is still 100,and again,what happend if we use the game.respawn_player() function,even if we use the if game.is_player_dead() as above it to judge,the code still block there and can not goes.can you give some advice
